src/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -1022,7 +1026,7 @@ public sealed partial class ImmutableStack<T> : System.Collections.Generic.IEnum
         public System.Collections.Immutable.ImmutableStack<T> Clear() { throw null; }
         public System.Collections.Immutable.ImmutableStack<T>.Enumerator GetEnumerator() { throw null; }
         public T Peek() { throw null; }
-#if FEATURE_ITEMREFAPI
+#if !NETSTANDARD10
```

Comment:
I think you're missing to define this constant when `TargetGroup == netstandard1.0`

Reply:
Maybe define this in corefx root `Directory.Build.props` and cleanup one other project which defines it. I suggest it that way because you will need to define it in both src and ref.

Something like: https://github.com/dotnet/corefx/blob/7373fd9794c4b52c2a4d253cc2bc4b47c2054241/Directory.Build.props#L417